src/sv-pipeline/04_variant_resolution/scripts/split_variants.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python
+import argparse
+import logging
+
+
+# Function to process the bed file by checking for conditions
+def process_bed_file(input_bed, n_per_split, bca=True):
+    svtype_field = 4
+    end_pos = 2
+    start_pos = 1
+
+    # Dictionary to store the conditions to be checked with matching prefixes
+    condition_prefixes = {
+        'gt5kb': {
+            'condition': lambda curr_1: (curr_1[svtype_field] == 'DEL' or curr_1[svtype_field] == 'DUP') and (int(curr_1[end_pos]) - int(curr_1[start_pos]) >= 5000)},
```

Comment:
I think this used to be `lambda line`, didn't it? I think that's nicer because it's clear what the lambda is operating on - `curr_1` is not very descriptive and it's unclear why there are curr_1, curr_2, and curr_3

Reply:
It was, once again PyCharm in fixing linting errors did not like it but I reverted it back to line.